hack/dind-cluster.sh

```diff
@@ -243,17 +252,28 @@ function enable-local-registry() {
     ${DOCKER_CMD} push ${ip}:${REGISTRY_PORT}/${image}:${tag}
   done
 
-  local image_prefix=$(echo ${images[0]} | cut -d'-' -f1)
-  echo "OPENSHIFT_IMAGE_FORMAT=${ip}:${REGISTRY_PORT}/${image_prefix}-\\\${component}:\\\${version}" >> "${config_root}/dind-env"
+  local error=0
+  $( grep -q "OPENSHIFT_IMAGE_FORMAT" "${config_root}/dind-env" ) || error=1
```

Comment:
What is the command substitution for? Seems like it should be `grep ... || error=1` without the `$()`.

I'd also forego the `error` variable as follows:
```bash
if ! grep -q -e 'OPENSHIFT_IMAGE_FORMAT' -- "${config_root}/dind-env"
then
  ...
fi
```
But if using the `error` variable is more readable, then that's fine.